q2_feature_classifier/_cutter.py

```diff
@@ -203,5 +227,11 @@ def extract_reads(sequences: DNAIterator,  f_primer: str, r_primer: str,
                                           'before trimming and truncation, '
                                           'so plan accordingly. Set to zero '
                                           '(default) to disable max length '
-                                          'filtering.'}
+                                          'filtering.',
+                            'n_jobs': 'Number of seperate processes to run. '
+                                      'Default is one.',
```

Comment:
No need to mention the default value here --- interfaces should indicate the default if it exists.